packages/bun-usockets/src/bsd.c

```diff
@@ -446,61 +488,47 @@ LIBUS_SOCKET_DESCRIPTOR bsd_create_listen_socket(const char *host, int port, int
 
     LIBUS_SOCKET_DESCRIPTOR listenFd = LIBUS_SOCKET_ERROR;
     struct addrinfo *listenAddr;
-    for (struct addrinfo *a = result; a && listenFd == LIBUS_SOCKET_ERROR; a = a->ai_next) {
+    for (struct addrinfo *a = result; a != NULL; a = a->ai_next) {
         if (a->ai_family == AF_INET6) {
             listenFd = bsd_create_socket(a->ai_family, a->ai_socktype, a->ai_protocol);
             listenAddr = a;
+
+            if (listenFd == LIBUS_SOCKET_ERROR) {
+                break;
```

Comment:
I think this break and lines 505, 517, and 525 should be `continue` so we can try more entries in `result`

Reply:
done